spec-dtslint/operators/publish-spec.ts

```diff
@@ -0,0 +1,16 @@
+import { of, Observable } from 'rxjs';
+import { publish } from 'rxjs/operators';
+
+it('should support empty parameter', () => {
+  // Typescript limitations - inference only happen in one direction from operator to pipe
+  const a = of(1, 2, 3).pipe(publish()); // $ExpectType Observable<any>
```

Comment:
I think you should also include const `a = of(1, 2, 3).pipe(publish<number>());` to show that `Observable<number>` is inferred if the type parameter is specified. This shows that inference works in one direction, despite `publish` returning `UnaryFunction<Observable<T>, ConnectableObserable<T>>` instead of  `UnaryFunction<Observable<T>, Obserable<T>>`.

Reply:
done :)